components/filter/filter-overflow-group.js

```diff
@@ -1,24 +1,62 @@
-import { css, LitElement } from 'lit';
-import { OverflowGroupMixin } from '../overflow-group/overflow-group-mixin.js';
+import './filter.js';
+import { css, html, LitElement } from 'lit';
+import { OVERFLOW_CLASS, OverflowGroupMixin } from '../overflow-group/overflow-group-mixin.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+function createFilterItem(node) {
+	const dimensionSets = node.querySelectorAll('d2l-filter-dimension-set');
+	const clones = Array.from(dimensionSets).map((set) => set.cloneNode(true));
+	return clones;
+}
 
 /**
  * A component that can be used to display a group of filters that will be put into an overflow filter when they no longer fit on the first line of their container
  * @slot - d2l-filters to be added to the container
 */
-class FilterOverflowGroup extends OverflowGroupMixin(LitElement) {
+class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 
 	static get styles() {
 		return [super.styles, css`
 			::slotted(d2l-filter) {
 				margin-right: 0.3rem;
 			}
+			:host([dir="rtl"]) ::slotted(d2l-filter) {
+				margin-left: 0.3rem;
+				margin-right: 0;
+			}
 		`];
 	}
 
-	convertToOverflowItem() {
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		this.addEventListener('d2l-filter-change', this._handleFilterChange);
+	}
+
+	convertToOverflowItem(node) {
+		const tagName = node.tagName.toLowerCase();
+		if (tagName === 'd2l-filter') return createFilterItem(node);
+	}
+
+	getOverflowContainer(overflowItems) {
+		return html`
+			<d2l-filter class="${OVERFLOW_CLASS}" @d2l-filter-change="${this._handleFilterChange}">
+				${overflowItems}
+			</d2l-filter>
+		`;
 	}
 
-	getOverflowContainer() {
+	_handleFilterChange(e) {
```

Comment:
This function is to keep the `selected` state in sync whether selected in the slotted `d2l-filter` or in the overflow `d2l-filter`